regulations/static/regulations/css/less/module/paragraph-markers.less

```diff
@@ -59,5 +60,16 @@ ol {
 }
 
 .level-2 > li > p {
-    margin-top: 0;
+    margin-top: 1em;
+}
+
+// move key term headings up to center with paragraph markers
+.paragraph-marker + .key-term {
+  float: left;
+  margin-top: -5px;
```

Comment:
This implicitly assumes a line height, no? Can this be fixed without hard-coding this value?